content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -118,30 +124,30 @@ to manage the lifecycle of Istio.
 
 ## Uninstall
 
-* For option 1, uninstall using `kubectl`:
+- For option 1, uninstall using `kubectl`:
 
     {{< text bash >}}
     $ kubectl delete -f $HOME/istio.yaml
     {{< /text >}}
 
-* For option 2, uninstall using Helm:
+- For option 2, uninstall using Helm:
 
-    {{< tip >}}
-    Uninstalling this chart does not delete Istio's registered CRDs. Istio by design expects
-    CRDs to leak into the Kubernetes environment. As CRDs contain all runtime configuration
-    data in custom resources the Istio designers feel it is better to explicitly delete this
-    configuration rather then unexpectedly lose it.
-    {{< /tip >}}
+    {{< warning >}}
```

Comment:
Please place the warning either before the step or after the commands. Having it between the step and the commands interrupts the expected flow.